build.gradle

```diff
@@ -312,7 +312,7 @@ dependencyManagement {
 dependencies {
 
     implementation group: 'com.github.hmcts', name: 'et-common', version:'1.0.6'
-    implementation group: 'com.github.hmcts', name: 'et-data-model', version:'1.1.2'
+    implementation group: 'com.github.hmcts', name: 'et-data-model', version:'1.1.4'
```

Comment:
tag is v1.2.0 and here 1.1.4?

Reply:
Also a small thing to take care of: Creating tag in et-data-model before merging it into master may sometimes lead to conflict